doc/data-files-description.md

```diff
@@ -81,13 +81,13 @@ This document contains information about all data files associated with this pro
-|`whole_exome_agilent_plus_tcga_6k.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg19 WXS regions provided by the kit manufacturer; obtained from the GDC website
\ No newline at end of file
+|`intersected_whole_exome_agilent_designed_120_AND_tcga_6k_genes.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_designed_120.targetIntervals.Gh38.bed`
+|`intersected_whole_exome_agilent_plus_tcga_6k_AND_tcga_6k_genes.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | Generated using bedtools intersect from `tcga_6k_genes.targetIntervals.Gh38.bed` and `whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed`
+|`tcga_6k_genes.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg38 version of `tcga_6k_genes.targetIntervals.bed`.  Generated using CrossMap and bedtools sort and merge. Script in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/CrossMap.sh)
+|`tcga_6k_genes.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from GDC website
+|`whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.Gh38.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg38 version of `whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed`. Generated using CrossMap and bedtools sort and merge. Script in [TCGA-capture-kit-analyses](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/tcga-capture-kit-investigation/scripts/CrossMap.sh)
+|`whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.bed` | Reference Target/Baits File | [SNV and INDEL calling](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/03.methods.md#snv-and-indel-calling) | hg19 WXS target capture regions downloaded from GDC website
```

Comment:
An exact link to where these files are available at each of these GDC website references would be helpful!